### PR TITLE
added changelog and add new test button to 3 pages

### PR DESCRIPTION
**Pull request by ma-zah**
pls review and merge, add if I missed something - there was some talk in the release notes about selectors, not sure if relevant and how to describe that change

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,38 @@ description: "All the nice stuff you asked for"
...
+<Frame caption="Element that was interacted with is highlighted by a bounding box, screenshot 01/2024">
+  <img
+    src="images/changelog/bounding-box.png"
+    alt="bounding boxes in test steps screenshot"
+  />
+</Frame>
+
+## 2023-12-18
+
+- Hanging tests runs mitigated
```
Comment (Germandrummer92) [inline changelog.mdx:21]:
```suggestion
- Fixed: Sometimes some of our test runs got stuck, we improved this and no stuck tests should occurr anymore.
```

File: changelog.mdx [hunk 2]
```diff
@@ -4,6 +4,38 @@ description: "All the nice stuff you asked for"
...
+
+## 2023-12-13
+
+- Delete steps option in a test case draft added
+- Idle `AI agent`: our agent will let you know if it's idle and waiting for more input
+
+## 2023-12-11
+
+- You can edit test cases, now. Delete steps in test cases or mark them as optional, so their failure is ignored. Enter text parameters. Create test case drafts and publish them. Archive test cases. Create new drafts from archived tests when needed.
```
Comment (Germandrummer92) [inline changelog.mdx:30]: delete is now mentioned teice here and above :)

Comment (Germandrummer92) [inline changelog.mdx:30]:
```suggestion
- You can edit test cases, now. Mark steps as optional, so their failure is ignored. Adjust text parameters for steps entering text or selecting a menu item. You can create test case drafts and publish them.
```

File: changelog.mdx [hunk 3]
```diff
@@ -4,6 +4,38 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
```
Comment (Germandrummer92) [inline changelog.mdx:4]: and yes selector editing is relevant: "You can now edit the playwright selectors that are used to find the element in question for the step"

**Review by Germandrummer92 — APPROVED** [review]

File: changelog.mdx (hunk 3 above)
Comment (ma-zah) [reply changelog.mdx:4]: ok, I'll add it

File: changelog.mdx (hunk 1 above)
Comment (Germandrummer92) [reply changelog.mdx]: I think i misspelled occur I think only one r, sorry 😬

Comment (ma-zah) [reply changelog.mdx]: that's what happens when you listen to Cardi B for too long

**Pull request merged**